src/screens/Login/types.tsx

```diff
@@ -0,0 +1,9 @@
+export interface LoginState {
+  email: string;
+  password: string;
+  isLoading: boolean;
+}
+
+export interface LoginProps {
```

Comment:
Same here `ILoginProps`. Please apply this convention for the rest of your typescript definition's name.